coalib/coala_main.py

```diff
@@ -183,7 +183,8 @@ def run_coala(console_printer=None,
                 section['default_actions'] = '*: ShowPatchAction'
                 section['show_result_on_top'] = 'yeah'
 
-            print_section_beginning(section)
+            if not (section.name == 'cli' and len(section.contents) == 0):
```

Comment:
The above approach wouldn't work.
The reason being `section.contents` is never empty. Even if your `.coafile` is something like below where your [cli] section is empty. 
```
[cli]

[python]
files = admin.py
bears = SpaceConsistencyBear, QuotesBear
language = Python
preferred_quotation = '

default_actions = **: ApplyPatchAction
```
When i tried to debug it, `section.contents` always returns a `Ordered dict` of length 3 when the contents in the [ cli ] section is empty. I tried to print the value of `section.contents` when the [ cli ] section is empty. It's something like:- 
```
OrderedDict([('comment0', <Setting object(key='comment0', value='', origin='/home/theprophet/git/thepursuit/oth/.coafile', from_cli=False, to_append=False) at 0x7f5ed01aa1d0>)])

```
From the above result I can positively say that section always maintains some value in its contents which it uses for the proper working of coala. 
That's the reason, you checking if `len(section.contents) == 0` will not work. Since this doesn't work your `if` condition fails and the message `Executing Section CLI` will always prompt up.

I guess you need to change your approach.

P.S:- Please test your changes locally before creating a PR. This will save yours and the reviewers time :)

Reply:
> OrderedDict([('comment0', <Setting object(key='comment0', value='', origin='/home/theprophet/git/thepursuit/oth/.coafile', from_cli=False, to_append=False) at 0x7f5ed01aa1d0>)])

Yeah this true when `debug=True`, but when `debug=False` output is `OrderedDict()`.